sway-core/src/language/asm.rs

```diff
@@ -10,6 +10,17 @@ pub struct AsmOp {
     pub(crate) immediate: Option<Ident>,
 }
 
+impl AsmOp {
+    pub fn retd(ptr: BaseIdent, len: BaseIdent) -> Self {
```

Comment:
Is this function ever used? If yes, do we always need to lose span information?

Reply:
We don't need these function at the moment. I will remove them.